contributing/development/compiling/compiling_for_windows.rst

```diff
@@ -132,7 +132,9 @@ Running SCons
 ~~~~~~~~~~~~~
 
 After opening a command prompt, change to the root directory of
-the engine source code (using ``cd``) and type::
+the engine source code (using ``cd``) and type
```

Comment:
```suggestion
the engine source code (using ``cd``) and type:
```
I'd keep the colon here, same everywhere (as it's printed as a single colon when using `::`)